pkg/server/types.go

```diff
@@ -39,7 +39,20 @@ type BackendMapper struct {
 	currentModes string
 }
 
+// Meta is the collection of fields which should be included in all top-level
+// objects which are propagated for dynamic file mode
+type Meta struct {
+	APIVersion string `json:"ApiVersion"`
```

Comment:
how is APIVersion used? Would it have constraint on being a semver?

Reply:
As of right now, it is not being used. It would have a constraint on being semvar though.